arraylist/src/main/java/ua/hillel/ArrayList.java

```diff
@@ -1,68 +1,123 @@
 package ua.hillel;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 /**
  * Implement a list that is based on array. This resizable data structure
  * is based on an array and is a simplified version of {@link java.util.ArrayList}.
  */
 public class ArrayList<T> implements List<T> {
+    private Object[] arrayElements;
```

Comment:
There is no need to point out these are the array elements. You can rename it to `elements`.